composer.lock

```diff
@@ -73,7 +73,7 @@
                 "docblock",
                 "parser"
             ],
-            "time": "2015-08-31 12:32:49"
+            "time": "2015-08-31T12:32:49+00:00"
```

Comment:
ok, thus :+1:
(on a side not, that approach would require this demo project to update lock file on regular basis)